package.json

```diff
@@ -36,7 +36,7 @@
     "kerberos": "^2.0.1",
     "mongodb-client-encryption": ">=6.0.0 <7",
     "snappy": "^7.2.2",
-    "socks": "^2.7.1"
+    "socks": "^2.7.3"
```

Comment:
This is a change to the peer deps, we can only change it in [dev deps](https://mongodb.slack.com/archives/GGWBN4ZNK/p1708028241681019?thread_ts=1708025612.915399&cid=GGWBN4ZNK).